x-pack/plugins/reporting/server/core.ts

```diff
@@ -126,21 +126,10 @@ export class ReportingCore {
     const config = createConfig(core, context.config.get<ReportingConfigType>(), logger);
     this.config = config;
 
-    // Export Type declarations
-    this.exportTypes.push(
-      new CsvSearchSourceExportType(this.core, this.config, this.logger, this.context)
-    );
-    this.exportTypes.push(new CsvV2ExportType(this.core, this.config, this.logger, this.context));
-    this.exportTypes.push(new PdfExportType(this.core, this.config, this.logger, this.context));
-    this.exportTypes.push(new PngExportType(this.core, this.config, this.logger, this.context));
-    // deprecated export types for tests
-    this.exportTypes.push(new PdfV1ExportType(this.core, this.config, this.logger, this.context));
-    this.exportTypes.push(new PngV1ExportType(this.core, this.config, this.logger, this.context));
-
+    this.exportTypes = this.validateExportTypesWithConfig();
```

Comment:
This function name seems pretty specific to how it's implemented. I recommend using a generic name that would still make sense even if the implementation was different, and simple like `getExportTypes`

Reply:
Cool! Changed in [6d4273b](https://github.com/elastic/kibana/pull/162815/commits/6d4273b8b051db14044c51d821bea2bba69af627)